app/subscriber/src/components/content-list/ViewOptions.tsx

```diff
@@ -1,13 +1,52 @@
 import { TooltipMenu } from 'components/tooltip-menu';
 import React from 'react';
 import { FaGear } from 'react-icons/fa6';
-import { Checkbox, Col, Radio, Row } from 'tno-core';
+import { useUsers } from 'store/hooks';
+import { useAppStore } from 'store/slices';
+import { Checkbox, Col, IUserModel, Radio, Row } from 'tno-core';
 
 import { ContentListContext } from './ContentListContext';
 import * as styled from './styled';
 
 export const ViewOptions: React.FC = () => {
   const { viewOptions, setGroupBy, setViewOptions, groupBy } = React.useContext(ContentListContext);
+  const [{ userInfo }, store] = useAppStore();
+  const api = useUsers();
+
+  /** Save the user's preferences for view/grouping options under preferences */
+  const savePreferences = async () => {
+    if (userInfo) {
+      const user = {
+        ...userInfo,
+        preferences: { ...userInfo.preferences, viewOptions, groupBy },
+        isSystemAccount: false,
```

Comment:
I notice we hardcoded these values in `MyMinisterSettings` as well, should I add an option to `findUser` in the subscriber app so we can avoid potentially overriding these values?

Reply:
Not sure if this would pose further implications security wise, but they are currently required fields when updating the user.

Reply:
We shouldn't let a subscriber update these fields.  As for the model, it's a generic one we're using across the API.  You could create an appropriate model for the subscriber endpoint.

Reply:
I'll make a model!